wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -408,7 +408,10 @@ extension AppRootRouter {
         userSession: UserSession,
         isComingFromRegistration: Bool
     ) -> AuthenticatedRouter? {
-        let isTeamMember: Bool
+        let needToShowDataUsagePermissionDialog = appStateCalculator.wasUnauthenticated && !SelfUser.current.isTeamMember
+        guard let userSession = ZMUserSession.shared() else { return  nil }
+
+      let isTeamMember: Bool
```

Comment:
`isTeamMember` is duplicated now and also `!SelfUser.current.isTeamMember` doesn't exist anymore, it's now `SelfUser.provider?.providedSelfUser`. I guess the logic here must be updated to work, maybe it was a merge conflict?

Reply:
Fixed it!